runtime/runtime.go

```diff
@@ -86,24 +84,27 @@ func enable(name, inTerms string) error {
 		fmt.Printf("failed to enable \"%s=%s\" (%v)\n", name, inTerms, err)
 		return err
 	}
+
+	fp.failpointMu.Lock()
+	defer fp.failpointMu.Unlock()
+
 	fp.t = t
```

Comment:
```suggestion
	fp.SetTerm(t)
```
Adding a lock is good, however let's not depend on caller using it correctly. Rather wrap access to failpoint with a methods.

Reply:
Thanks for the suggestion @serathius! 

I will merge these commits and double-check the work! :)